README.md

```diff
@@ -397,3 +397,195 @@ const UserForm = () => (
   </div>
 );
 ```
+
+&nbsp;
+
+---
+
+&nbsp;
+
+## FAQ
+
+> ### When should I use `createKeyedResource` vs `createSingleEntryResource`?
+
+The the main difference between `createKeyedResource` and `createSingleEntryResource` is how the created `resource` stores its entries. Internally, every `resource` organizes the data it receives from the completed load function into entries. A `resource`'s main concern is how the data in the entries should be available to your app, not necessarily how the external api is structured or called.
+
+- `createKeyedResource` creates a resource with multiple entries that are organized by key. [See the first parameter on how entries are keyed](https://github.com/chadwatson/react-remote-resource#createkeyedresource).
+
+- `createSingleEntryResource` creates a resource with only a single entry. All data, regardless of the structure, will be stored in only one entry.
+
+Here are some examples scenerios that show when each of the :
+
+#### Example Users API:
+
+Assuming You have a users api that takes an id and returns user information:
+
+```javascript
+/*
+  
+  /api/users/:id
+
+  Example Response:
+  {
+    name: "name",
+    id: {userId},
+    ...
+  }
+
+*/
+```
+
+**Scenerio 1:**
+
+The app only needs the current users information. Since the app only one entry from the api (the current users information and no others), `createSingleEntryResource` would work well.
+
+```jsx
+const load = (id) => fetch(`/api/users/${id}`);
+const userResource = createSingleEntryResource(load);
+
+const currentUserId = ...;
+
+const AboutMe = () => {
+  const [user] = userResource.useEntry(currentUserId);
+
+  return ...;
+}
```

Comment:
Might be clearer to just supply a hard-coded ID to `useEntry`.

```jsx
const load = (id) => fetch(`/api/users/${id}`);
const userResource = createSingleEntryResource(load);

const AboutMe = () => {
  const [user] = userResource.useEntry(12345);
  return ...;
};
```

Reply:
Yes it is!